src/zenml/zen_server/routers/users_endpoints.py

```diff
@@ -460,6 +513,15 @@ def update_myself(
         Returns:
             The updated user.
         """
+        current_user = zen_store().get_user(auth_context.user.id)
+        user.activation_token = current_user.activation_token
+        user.active = current_user.active
+
+        if user.is_admin:
```

Comment:
```suggestion
        if user.is_admin is not None:
```

Reply:
Not needed with your previous suggestion to keep fixed `is_admin` in update self